docs/reference/handlers/0-base-handler.md

```diff
@@ -1,3 +1,6 @@
 # Base Route Handler
 
 ::: starlite.handlers.base.BaseRouteHandler
+    options:
+        members:
+            - opt
```

Comment:
The doc accesses `opt` multiple times, for example [here](https://starlite-api.github.io/starlite/usage/9-guards/#the-route-handler-opt-key) so it should be exposed in the reference or BaseRouteHandler